### PR TITLE
disable nic_hotplug/unplug for spapr-vlan bus

### PR DESCRIPTION
**Pull request by vipmike007**
Currently, spapr-vio bus doesn't support hot pluging.
So variant spapr-vlan shouldn't be added to this case
This patch is to remove it

ID: 1214221

Signed-off-by: Mike Cao <bcao@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
LGTM,

Acked-by: Xu Tian <xutian@redhat.com>

**Comment by vipmike007** [comment]
closing for it will supported in the future.

**Comment by vipmike007** [comment]
After talking w/ feature owner this patch is still valid to ppc.

File: qemu/tests/cfg/nic_hotplug.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 - nic_hotplug: install setup image_copy unattended_install.cdrom
-    no RHEL.3
+    no RHEL.3
```
Comment (suqinhuang) [inline qemu/tests/cfg/nic_hotplug.cfg]: Hi bcao,
Could you check this line

**Comment by XueqiangWei** [comment]
LGTM,
Acked by Xueqiang Wei <xuwei@redhat.com>

**Comment by xutian** [comment]
Merged. thanks @vipmike007

**Pull request merged**